contracts/package.json

```diff
@@ -33,7 +33,7 @@
     "shelljs": "^0.8.4",
     "solhint": "^3.3.4",
     "solhint-plugin-prettier": "^0.0.5",
-    "solidity-bytes-utils": "0.8.0",
+    "solidity-bytes-utils": "https://github.com/GNSPS/solidity-bytes-utils#feat/update-0.7.0",
```

Comment:
Ah we can remove this dependency now, as the Uniswap module already has an 0.7.x compatible version of this

Reply:
alright cool, and yeah I couldn't find the version I needed in the npm / yarn registry (so got it from github)